### PR TITLE
VRF-LITE support ARP

### PR DESCRIPTION
**Pull request by Tristan-Labbe**
New pull request with only arp-table discovery.
For more details, you can see the pull https://github.com/librenms/librenms/pull/3255

### CODE REVIEW

**Comment by Tristan-Labbe** [comment]
Hello @laf,

Can you say me if this PR is OK ?

**Comment by laf** [comment]
Seems fine, I assume it relies on #3255 though?

**Comment by Tristan-Labbe** [comment]
Yes it relies on https://github.com/librenms/librenms/pull/3255.

**Comment by Tristan-Labbe** [comment]
What is the next step ?

**Comment by Tristan-Labbe** [comment]
Up. 
Please can you tell me what is the next step for this PR ?

**Comment by laf** [comment]
The other PR needs to be fixed so we can merge if this relies on it.

File: includes/discovery/arp-table.inc.php [hunk 1]
```diff
@@ -4,6 +4,16 @@
...
 echo 'ARP Table : ';
 
+if( key_exists('vrf_lite_cisco', $device) && (count($device['vrf_lite_cisco'])!=0) ){
+    $vrfs_lite_cisco = $device['vrf_lite_cisco'];
+}
+else  {
+    $vrfs_lite_cisco = array(array('context_name'=>null));
+}
+
+foreach ($vrfs_lite_cisco as $vrf) {
```
Comment (laf) [inline includes/discovery/arp-table.inc.php]: This seems to be the same as the other PR.

I'm not convinced this code does what is intended. You are setting $vrfs_lite_cisco to a string if vrf_lite_cisco is detected and then trying to loop through it. If it's not found vrf_lite_cisco then your setting an array into an array and trying to loop through it.

**Pull request merged**